lisa/operating_system.py

```diff
@@ -1814,6 +1814,35 @@ def _update_packages(self, packages: Optional[List[str]] = None) -> None:
         if self._no_repo_enabled.search(output):
             raise RepoNotExistException(self._node.os)
 
+    def grow_os_partition_size(self) -> None:
+        from lisa.tools import Mount
+
+        pv_result = self._node.execute("pvscan -s", sudo=True, shell=True).stdout
+        matched = re.compile(r"(?P<disk>.*)(?P<number>[\d]+)", re.M).match(
```

Comment:
Use LISA built-in methods from utils for capture groups.

Reply:
updated